ouroboros-consensus/src/ouroboros-consensus/Ouroboros/Consensus/Storage/ChainDB/API.hs

```diff
@@ -380,6 +380,9 @@ data ChainDB m blk = ChainDB {
                 (PointNotFound blk)
                 (LedgerTables (ExtLedgerState blk) ValuesMK))
 
+
+    , getCurrentLedgerTableSize :: m (Either (WithOrigin SlotNo, WithOrigin SlotNo) Int)
```

Comment:
This needs some documentation

Reply:
Indeed. If possible I think it's better to add it in this PR, otherwise, it'll become more difficult the more time it passes.

Reply:
Done